src/main/java/hudson/plugins/parameterizedtrigger/TriggerBuilder.java

```diff
@@ -133,9 +134,11 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher,
                     for (Job p : projectList) {
                         //handle non-buildable projects
                         if(!config.canBeScheduled(p)){
+                            User user = User.current();
+                            String userName = user != null ? ModelHyperlinkNote.encodeTo(user) : "unknown";
                             listener.getLogger().println("Skipping " + HyperlinkNote.encodeTo('/'+ p.getUrl(), p.getFullDisplayName()) + 
                                     ". The project is either disabled,"
-                                    + " or the authenticated user " + ModelHyperlinkNote.encodeTo(User.current()) + " has no Item.BUILD permissions,"
+                                    + " or the authenticated user " + userName + " has no Item.BUILD permissions,"
```

Comment:
"the authenticated user unknown has no ..." is a bit odd, but should be ok

Reply:
It should be `the authenticated user unknown has no Item.BUILD`